### PR TITLE
Fixed the links of stable release

### PR DESCRIPTION
**Pull request by daveyheuser**
## What are your changes?
Fixed the links of stable release

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [ ] Medium

## I have completed these items:
- [ ] I have built locally and tested for broken links and formatting.
- [ ] If appropriate, I have added redirects.

### CODE REVIEW

File: src/releases/index.jade [hunk 1]
```diff
@@ -26,9 +26,9 @@ block content
                 a.btn.btn-primary.mb0.block.center.no-arrow.text-white.bg-heliotrope Get Release
                   img.ml1.align-middle(src='/assets/images/icons/arrow-down-docs-unselected-white.svg' style='margin-top: -2px;')
                 ul
-                  li: a(href='https://dcos.io/docs/1.8/administration/installing/cloud/aws/') Amazon Web Services
+                  li: a(href='https://downloads.dcos.io/dcos/stable/aws.html?_ga=1.40383586.1790439002.1478539864') Amazon Web Services
                   li: a(href='https://dcos.io/docs/1.8/administration/installing/cloud/azure/') Azure
```
Comment (judithpatudith) [inline src/releases/index.jade]: This should link to https://downloads.dcos.io/dcos/stable/azure.html?_ga=1.99539678.1790439002.1478539864

File: src/releases/index.jade [hunk 2]
```diff
@@ -26,9 +26,9 @@ block content
...
-                  li: a(href='https://downloads.dcos.io/dcos/stable/dcos_generate_config.sh?_ga=1.191046122.1509614140.1478184618') Direct Download
```
Comment (judithpatudith) [inline src/releases/index.jade]: This should have stayed the same

**Comment by daveyheuser** [comment]
Like this?

File: src/releases/index.jade (hunk 1 above)
Comment (joel-hamill) [inline src/releases/index.jade]: this portion of URLs is not needed `?_ga=1.99539678.1790439002.1478539864` - this is added dynamically and is a part of tracking

**Comment by judithpatudith** [comment]
Fixed by @karlkfi during 1.9 EA release

**Pull request closed without merging**